jobs/ci-kubernetes-verify-release-1.5.sh

```diff
@@ -25,7 +25,8 @@ export KUBE_VERIFY_GIT_BRANCH='release-1.5'
 export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
 
 ### Runner
-timeout -k 15m 80m "${KUBE_TEST_SCRIPT}" && rc=$? || rc=$?
+readonly runner="./hack/jenkins/gotest-dockerized.sh"
+timeout -k 15m 80m "${runner}" && rc=$? || rc=$?
```

Comment:
I'm not sure I understand. Does `KUBE_TEST_SCRIPT` affect what happens in `gotest-dockerized.sh`?